gradio/components/image.py

```diff
@@ -25,6 +25,17 @@
 PIL.Image.init()  # fixes https://github.com/gradio-app/gradio/issues/2843
 
 
+class ImageData(GradioModel):
```

Comment:
This can just inherit from FileData and add the new b64 field?

Reply:
Actually, do we even need a new b64 field? Can we just use the `url` field for this?

Reply:
As discussed, I think it would be better to have a separate dataclass for base64-formatted data as most of the other fields are not relevant in the base64 case.